addons/web/static/src/views/relational_model.js

```diff
@@ -2432,6 +2429,20 @@ export class DynamicGroupList extends DynamicList {
         };
         return new FormArchParser().parse(quickCreateForm.arch, models, this.modelName);
     }
+
+    /**
+     * @param {Object} groupData
+     * @returns {DateTime | false}
+     */
+    _parseDateValue(groupData) {
```

Comment:
not sure about this function name, it sounds very generic, but as far as I understand, the code is really specific